roles/foreman_provisioning/tasks/configure_centos_9.yml

```diff
@@ -0,0 +1,27 @@
+- name: "Ensure CentOS Stream 9"
+  theforeman.foreman.operatingsystem:
+    username: "{{ foreman_provisioning_username }}"
+    password: "{{ foreman_provisioning_password }}"
+    server_url: "{{ foreman_provisioning_server_url }}"
```

Comment:
Instead of having to pass username/password/url to every module, Ansible supports so-called module-defaults.

However, you can only define those in playbooks, not in roles (don't ask), which would mean we need something like

```
- hosts: all
  become: true
  vars:
    libvirt_tftp: true
  module_defaults:
    group/theforeman.foreman.foreman:
      username: admin
      password: changeme
      server_url: "https://localhost/"
      validate_certs: false
  roles:
    - foreman
    - libvirt
    - foreman_provisioning
```
for the `playbooks/foreman_provisioning.yml` playbook

Reply:
Looks like the linter doesn't understand that.

Reply:
@evgeni @ekohl to fix this what is the best approach to resolve this? Should I just add this in `.ansible-lint` skip_list?